pandas/core/indexes/range.py

```diff
@@ -496,6 +496,7 @@ def argsort(self, *args, **kwargs) -> npt.NDArray[np.intp]:
         numpy.ndarray.argsort
         """
         ascending = kwargs.pop("ascending", True)  # EA compat
+        kwargs.pop("kind", None)  # e.g. "mergesort" is irrelevant
```

Comment:
should we add to the signature? (as ignored but accepted args), I *think* we do this elsewheree

Reply:
the other Index subclasses all use `*args, **kwargs`